Package.swift

```diff
@@ -24,11 +24,34 @@
 
 import PackageDescription
 
-let swiftSettings: [SwiftSetting] = [
-    .define("CRYPTO_IN_SWIFTPM"),
-    // To develop this on Apple platforms, uncomment this define.
-    // .define("CRYPTO_IN_SWIFTPM_FORCE_BUILD_API"),
-]
+// To develop this on Apple platforms, set this to true
+let development = false
```

Comment:
Should we give a way of setting this without changing the code?  Eg env var or something?

Reply:
Given that the only intention is to make this available when developing locally, structuring things like this is fine.